lib/octicons_react/pages/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import {Block, Text} from 'primer-react'
 import pkg from '../package.json'
-import Octicon, {iconsByName} from '../'
+import Octicon, {iconsByName} from '../index.umd'
```

Comment:
I thought we needed to remove `iconsByName`?

Reply:
It is still there in the UMD bundle, since that contains all the icons anyway we might as well keep it around! :blush: I can remove it if you prefer though.